src/Nimbus.Testing/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System.Reflection;
+using System.Runtime.InteropServices;
+
+// General Information about an assembly is controlled through the following
+// set of attributes. Change these attribute values to modify the information
+// associated with an assembly.
+[assembly: AssemblyTitle("Nimbus.Testing")]
+[assembly: AssemblyDescription("")]
+[assembly: AssemblyConfiguration("")]
+[assembly: AssemblyCompany("")]
+[assembly: AssemblyProduct("Nimbus.Testing")]
+[assembly: AssemblyCopyright("Copyright ©  2018")]
```

Comment:
Can you get the above filled out to be consistent with the other projects? Will help with acceptance of pull request to upstream.